ble_reflector.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import time
+import sys
+import subprocess
+from parameters import EEGInfo, SharedParameters
+from Mindwave.mindwave import Datapoint
+
+class RelaxReflector:
+    """Utility to output headset and game data over Bluetooth LE
+    """
+    params = None
+    # last_game_data_time = None
+
+    process = None
+
+    @classmethod
+    def create_reflector(self, params):
+        self.params = params
+        self.process = subprocess.Popen(['node', 'serial-central.js'], 
+            stdin=subprocess.PIPE)#, cwd='')
+
+    @classmethod
+    def shutdown_reflector(self):
+        self.process.communicate('q\n')
+        self.process.terminate()
+
+    @classmethod
+    def headset_data_changed(self, use_eeg2=False):
+        eeg = None
+        if use_eeg2 == True:
+            eeg = self.params.eeg1
+            self._send_data(player_one_data=eeg)
+        else:
+            eeg = self.params.eeg2
+            self._send_data(player_two_data=eeg)
+
+    @classmethod
+    def game_data_changed(self, game_time=None):
+        self._send_data(game_time=game_time)
+
+    @classmethod
+    def _send_data(self, player_one_data=None, player_two_data=None, game_time=None):
+        def eeg_to_string(eeg):
+            if eeg == None:
+                return '[]'
+            return '[{},{},{},{},{},{}]'.format(eeg.meditation, eeg.meditation_avg, eeg.attention, eeg.attention_avg, eeg.on, eeg.poor_signal)
```

Comment:
You're passing these strings into the Node process, I'm assuming it's expecting JSON? If so I might have a suggestion here.